beast/observationmodel/tests/test_toothpick_noisemodel.py

```diff
@@ -37,7 +37,8 @@ def test_toothpick_noisemodel():
                             asts_fname,
                             modelsedgrid,
                             absflux_a_matrix=absflux_a_matrix,
-                            vega_fname=vega_fname)
+                            vega_fname=vega_fname,
+                            use_rate=False)
```

Comment:
similar question for this - I'm guessing changing to `True` would mess with tests?

Reply:
Yep.